### PR TITLE
Homework#2 Narzędzia

### PR DESCRIPTION
**Pull request by Petrus9107**

### CODE REVIEW

File: src/functions.cpp [hunk 1]
```diff
@@ -1,10 +1,10 @@
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
```
Comment (Marcinati) [inline src/functions.cpp:1]: You have left unnecessary comment.

File: src/functions.cpp [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 #include <algorithm>
 #include <iostream>
 #include <limits>
 
 void print(std::vector<int> const & numbers) {
-    for (int i = 0; i < numbers.size(); ++i) {
+    for (unsigned int i = 0; i < numbers.size(); ++i) {
```
Comment (Marcinati) [inline src/functions.cpp:7]: The best options is to use std::size_t here.

File: src/main.cpp [hunk 3]
```diff
@@ -1,6 +1,6 @@
 #include <vector>
 #include <iostream>
-#include "../inc/functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
+#include "functions.hpp"    // TODO: add proper compilation flag and change to #include "functions.hpp"
```
Comment (Marcinati) [inline src/main.cpp:3]: Another unnecessary comment.

File: src/functions.cpp (hunk 2 above)
Comment (Marcinati) [inline src/functions.cpp:7]: The best options is to use std::size_t here.

File: src/functions.cpp (hunk 1 above)
Comment (Marcinati) [inline src/functions.cpp:1]: You have left unnecessary comment.

**Review by Marcinati — CHANGES_REQUESTED** [review]
Some changes.

File: src/functions.cpp (hunk 2 above)
Comment (ziobron) [reply src/functions.cpp:7]: Yup, size_t is just a universal type for any size. You do not have to worry if it's unsigned or unsigned long or anything else, so please always use size_t in such loops.

**Comment by ziobron** [comment]
+4 XP Narzędzia#2 – vimtutor
+5 XP Narzędzia#2 – git - konflikty
+4 XP Narzędzia#2 – flagi kompilacji
+3 XP Narzędzia#2 – git - cherrypick
+8 XP delivery before 07.06.2020

{ "requirement_ids": [1119, 1999, 1119, 2000, 1119, 2001, 1119, 2002], "points": [2, 4, 2, 5, 2, 4, 2, 3] }

**Pull request closed without merging**